server.js

```diff
@@ -1,12 +1,16 @@
 /**
  * Module dependencies.
  */
+
+// Require 'dotenv' config
+require('dotenv').config();
+
+
 var express = require('express'),
     fs = require('fs'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
-require('dotenv').config();
```

Comment:
I considered it would be good practice having the require statement for the config at the top of the script, I double checked after the modification to ensure there were no errors in the build after the change.